.github/workflows/ci-app-tests.yml

```diff
@@ -64,9 +64,11 @@ jobs:
           pip list
 
       - name: DocTests App
-        if: ${{ matrix.pkg-name != 'LAI' }}
         working-directory: src
-        run: python -m pytest lightning_app --ignore-glob="**/cli/*-template/**"
+        run: |
+          pip install pytest -q
```

Comment:
Why does this install pytest here? Shouldn't the "Install dependencies" below (L84) be merged with "Install package" above (L59)?

Reply:
Not really, we want to have a test if package is fine without installing al extras, so for a user who would just install PL/App it works, which was the case with Apps that it was not true and was crashing

Reply:
What does that have to do with doctests? Users would not run them

Maybe you want to do that instead in `test-pkg`: https://github.com/Lightning-AI/lightning/pull/15105/files#diff-f7f8305e901571c1ad55ba41890b6c6bb21aa4a1ba94b61ee77239860fe0bf1bR20 since we have it already

Reply:
yes we can run doctests in installation workflow...